### DISCUSSION TITLE
NET::ERR_CERT_AUTHORITY_INVALID

### DISCUSSION THREAD

**Discussion by igarcido95** [vercel/vercel#6237] [Help]
Hi,

One of our users is experiencing this SSL issue when try to access [our page](https://www.musiccareers.co/). I've checked the SSL setup and everything seems fine.

What information could I request the user to debug this issue?

<img width="816" alt="5b510772-25f1-41f9-ba75-8d44e82ab7dc" src="https://user-images.githubusercontent.com/42417619/118269085-8dc1b400-b4be-11eb-8490-82964fe8eabc.png">

Thanks!

**Comment by urjitbhatia** [comment]
You should ask them to open developer toolds and click on the `security` tab. This has an overview of the certificate. Get a screenshot of that and see if anything obvious jumps out. 
![Screen Shot 2021-05-14 at 6 29 23 PM](https://user-images.githubusercontent.com/1226567/118344292-51a04a80-b4e2-11eb-9d1b-b58fe3388fa0.png)
Secondly, get them to click on `view certificate`, expand all the details and send them to you.

Then look at what the certificate chain says - this error can mean a few things:

- The Certificate is self-signed.
- The Certificate was issued by an untrusted Certificate Authority (CA)
- Sometimes, when another domain like vercel issues a certificate on your behalf on a subdomain, the certificate authorities need to have the right configuration. CloudFlare calls it SSL for SaaS for example https://www.cloudflare.com/ssl-for-saas-providers/ Make sure your domain is compatible

When I open your page, I see the right certs. This user could be using something like Charles Proxy or something that is MITM'ing the certs - ask them if they have some tool like that. If they are, this could be the likely cause https://stackoverflow.com/questions/31281045/charles-proxy-causes-privacy-error

Hope this helps

**Comment by StephaneChevassus** [comment]
Hi @igarcido95, 

Sorry to bring this up a few years after... but we are going through the same issue. We have some clients that cannot access their sites (using custom domains on vercel) from their internal network. Yet they have firewall/filtering rules but the domain itself is not part of any rules.

The very weird part is that they can access Vercel site, or any *.vercel.app domain.

Although it has been a few years, I was wondering you remember how you dealt with this issue? (if you see ever see this message)

Thank you
